app/services/data_migrations/remove_incorrect_hesa_codes.rb

```diff
@@ -0,0 +1,55 @@
+module DataMigrations
+  class RemoveIncorrectHesaCodes
+    TIMESTAMP = 20210518233940
+    MANUAL_RUN = false
+
+    def change
+      ApplicationForm.where.not(equality_and_diversity: nil).find_each do |application_form|
+        remove_incorrect_codes_from(application_form)
+      end
+    end
+
+  private
+
+    def remove_incorrect_codes_from(application_form)
+      if disabilities_not_specified?(application_form) && hesa_disabilities_present?(application_form)
```

Comment:
Did this bug only affect changes from `value` to `prefer not to say`? Are there any situations where the candidate changed their answer from one to another and now the code is incorrect?

Reply:
They could have changed answer from _yes_ to _no_ in which case I think the bug still applies. The intention here is to pick up any cases where the candidate either said _no_, _Prefer not to say_ or the `disabilities` attribute is empty for some other reason.